products/green_fast_track/bash/build.sh

```diff
@@ -7,7 +7,7 @@ dbt deps
 dbt debug
 
 echo "Build seed tables"
-dbt build --select config.materialized:seed
+dbt build --select config.materialized:seed --indirect-selection=cautious --full-refresh
```

Comment:
what do these 2 arguments in the end do?

Reply:
`--full-refresh` ensures the entire table is rebuilt, not just values in existing columns. without it, this fails when there are new or different columns ([docs](https://docs.getdbt.com/faqs/Seeds/full-refresh-seed))

`--indirect-selection=cautious` ensures that when we say "test the seed tables" during `dbt build`, we don't run _all_ tests that depend on those tables ([docs](https://docs.getdbt.com/reference/node-selection/test-selection-examples?indirect-selection-mode=cautious#indirect-selection-examples)). in this case, the new test `green_fast_track_pilot_projects_equality` was being run before the table it's defined for exists)